primitives/primproc/columncommand.h

```diff
@@ -163,6 +173,8 @@ class ColumnCommand : public Command
   uint32_t baseMsgLength;
 
   uint64_t lbid;
+  bool _hasAuxCol;
```

Comment:
Naming style calls for an underscore for the class members.

Reply:
Done.